lib/generators/templates/setup/dev_prime.rb

```diff
@@ -0,0 +1,12 @@
+if Rails.env.development? || Rails.env.test?
+  require "factory_bot" if Bundler.rubygems.find_name("factory_bot_rails").any?
```

Comment:
I wonder if instead we conditionally should run `suspenders:factories` if the gem is missing?

Reply:
Can `suspenders:factories` add this file instead?

Reply:
I was toying with that idea when I made the `suspenders:factories` generator, but then the conditional logic would need to happen in the setup script.

```ruby
if File.exist? "lib/tasks/dev.rake"
  system! "bin/rails dev:prime"
else
    system! "bin/rails dev:setup"
end
```

I guess that's simpler, though.

Reply:
Or the `suspenders:factory` generator could `gsub("rails dev:setup", "rails dev:prime")`.

Reply:
I ended up opting for the conditional, since I don't like the idea of any other another generator affecting the setup script.

I'll need to either modify `suspenders:factory` to create the `dev:prime` task, or create a separate generator for this.

b074e46